### PR TITLE
[Improve] Improve read with parquet type convert error

### PR DESCRIPTION
**Pull request by Hisoka-X**
### Purpose of this pull request
This PR optimize the errors returned when reading unsupported field types in parquet files and provide more friendly and standardized error messages.

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
add new test.

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ParquetReadStrategy.java [hunk 1]
```diff
@@ -287,9 +299,8 @@ private SeaTunnelDataType<?> parquetType2SeaTunnelType(Type type) {
                         case DATE:
                             return LocalTimeType.LOCAL_DATE_TYPE;
                         default:
-                            String errorMsg = String.format("Not support this type [%s]", type);
-                            throw new FileConnectorException(
-                                    CommonErrorCodeDeprecated.UNSUPPORTED_DATA_TYPE, errorMsg);
+                            throw CommonError.convertToSeaTunnelTypeError(
+                                    "Parquet", type.toString(), name);
```
Comment (TaoZex) [inline seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ParquetReadStrategy.java]: How about declaring "Parquet" as a constant？

Comment (TaoZex) [reply seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ParquetReadStrategy.java]: It appears many times in the code.

Comment (Hisoka-X) [reply seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ParquetReadStrategy.java]: addressed.

**Review by TaoZex — APPROVED** [review]
LGTM

**Review by EricJoy2048 — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**